docs/static/spec/openapi/starter.yaml

```diff
@@ -0,0 +1,252 @@
+                        percent_of_cpu_time: 0.0
+                        state: timed_waiting
+                        path: /path/to/logstash-8.17.0/vendor/bundle/jruby/1.9/gems/puma-2.16.0-java/lib/puma/thread_pool.rb:187
+                        traces: java.lang.Object.wait(Native Method)", "org.jruby.RubyThread.sleep(RubyThread.java:1002)", "org.jruby.RubyKernel.sleep(RubyKernel.java:803)
+
+  /_health_report:
+    get:
+      summary: Get health status
+      description: >
+        The health API returns a report with the health status of Logstash and the pipelines that are running inside of it. 
+        The report contains a list of indicators that compose Logstash functionality.
+
+        Each indicator has a health status of: green, unknown, yellow, or red. 
+        The indicator provides an explanation and metadata describing the reason for its current health status.
+
+        The top-level status is controlled by the worst indicator status.
+
+        In the event that an indicator status is non-green, a list of impacts may be present in the indicator result which detail the functionalities that are negatively affected by the health issue. 
+        Each impact carries with it a severity level, an area of the system that is affected, and a simple description of the impact on the system.
+
+        Some health indicators can determine the root cause of a health problem and prescribe a set of steps that can be performed in order to improve the health of the system. 
+        The root cause and remediation steps are encapsulated in a diagnosis. 
+        A diagnosis contains a cause detailing a root cause analysis, an action containing a brief description of the steps to take to fix the problem, and the URL for detailed troubleshooting help.
+        
+        NOTE: The health indicators perform root cause analysis of non-green health statuses. 
+        This can be computationally expensive when called frequently.
+        
+      operationId: healthStatus
+      parameters:
+        - name: pretty
+          in: query
+          schema:
+            type: boolean
+          description: >
+            If you append `?pretty=true` to the request, the JSON returned will be pretty formatted. Use it for debugging only!
+      responses:
+        '200':
+          description: Indicates a successful call
+          content:
+            application/json:
+              examples:
+                healthStatusExample1:
+                  status: 
+                    - green: Logstash is healthy.
+                      unknown: Logstash health could not be determined.
+                      yellow: The functionality of Logstash is in a degraded state and may need remediation to avoid the health becoming red.
+                      red: Logstash is experiencing an outage or certain features are unavailable for use. 
+                  indicators: Information about the health of Logstash indicators. 
```

Comment:
The Health report API docs show hierarchical descriptions rather that examples. What's the best way to represent this type of content? Also, research nested format.